src/components/Place/PlaceCard/PlaceCard.tsx

```diff
@@ -73,14 +89,7 @@ export default React.memo(function PlaceCard(props: PlaceCardProps) {
         !loading && !place && "hidden",
       ])}
     >
-      <a
-        className="place-card__cover"
-        href={href}
-        onClick={(e: React.MouseEvent<HTMLAnchorElement>) => {
-          e.preventDefault()
-          href && navigate(href)
-        }}
-      >
+      <a className="place-card__cover" href={href} onClick={handleClickCard}>
```

Comment:
There is a component `<Link>` that has tracking embed in it (you can send tracking data as props) and navigate too